float32 speed   # x linear velocity
float32 yaw_rate # z angular velocity